java/core/src/test/org/apache/orc/TestMinSeekSize.java

```diff
@@ -51,11 +50,11 @@ public class TestMinSeekSize {
   private static FileSystem fs;
 
   private static final TypeDescription schema = TypeDescription.createStruct()
-    .addField("f1", TypeDescription.createLong())
-    .addField("f2", TypeDescription.createDecimal().withPrecision(20).withScale(6))
-    .addField("f3", TypeDescription.createLong())
-    .addField("f4", TypeDescription.createString())
-    .addField("ridx", TypeDescription.createLong());
+      .addField("f1", TypeDescription.createLong())
+      .addField("f2", TypeDescription.createDecimal().withPrecision(20).withScale(6))
+      .addField("f3", TypeDescription.createLong())
+      .addField("f4", TypeDescription.createString())
+      .addField("ridx", TypeDescription.createLong());
   private static final boolean[] AlternateColumns = new boolean[] {true, true, false, true, false
     , true};
```

Comment:
Just a question, is this line passed through checkstyle?

Reply:
Yes it does, it was complaining about indentation, hence just fixed it.

Does this look better?

```java
private static final TypeDescription schema =
      TypeDescription.createStruct()
                    .addField("f1", TypeDescription.createLong())
                    .addField("f2", TypeDescription.createDecimal().withPrecision(20).withScale(6))
                    .addField("f3", TypeDescription.createLong())
                    .addField("f4", TypeDescription.createString())
                    .addField("ridx", TypeDescription.createLong());
```